appsmithorg/appsmith
# feat: All outgoing requests should go through the configured proxy

Fixes #11749

## To test

1. Install `mitmproxy` with `brew install mitmproxy`.
2. Start the proxy with `mitmweb --listen-port 9020 --web-port 9021`.
4. Start Appmith with:
   ```
   docker run --name ace -p 8001:80 -v ~/appsmith-stacks:/appsmith-stacks -d --pull always -e HTTP_PROXY=http://host.docker.internal:9020 -e HTTPS_PROXY=http://host.docker.internal:9020 appsmith/appsmith-ce:release
   ```
5. Now launch Appsmith at http://localhost:8001, load the templates page, and check that a request to load templates from cloud-services shows up in the proxy's web UI.
6. Run any API request in Appsmith and see that that also appears in the proxy's web UI.
7. Try connect a git repo and check that git related queries are also made via the proxy.
